Automattic/vip-go-mu-plugins
# Add QM Panel to display info about js/css concatenation

## Description
Adds a QM panel for displaying debugging information about VIP's JS/CSS concatenation feature.

<img width="1323" alt="qm-concat-panel" src="https://github.com/Automattic/vip-go-mu-plugins/assets/251183/3477d8e4-a734-4e86-b502-7741ca7661b7">

Will not display any information without https://github.com/Automattic/nginx-http-concat/pull/77

## Changelog Description

Add "VIP JS/CSS Concat" Query Monitor Panel

### Added
- Add "VIP JS/CSS Concat" Query Monitor Panel

### Removed
- n/a

### Fixed
- n/a

### Changed
- n/a

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist


## Steps to Test

- See also https://github.com/Automattic/nginx-http-concat/pull/77. Helper plugin attached there, to ensure sufficient assets for logging.
- Without above, this PR will simply create a "VIP JS/CSS Concat" panel with no data (shows "No data logged" message)
